### PR TITLE
Pt locale

### PR DESCRIPTION
**Pull request by JCKodel**

### CODE REVIEW

File: lib/src/l10n/languages/croppy_pt.dart [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  @override
+  String get doneLabel => 'OK';
+
+  @override
+  String get materialFreeformAspectRatioLabel => 'Livre';
+
+  @override
+  String materialGetFlipLabel(LocalizationDirection direction) =>
+      'Flip to ${direction == LocalizationDirection.vertical ? 'vertical' : 'horizontal'}';
```
Comment (kekland) [inline lib/src/l10n/languages/croppy_pt.dart:29]: Seems to still be in English

Comment (JCKodel) [reply lib/src/l10n/languages/croppy_pt.dart:29]: Nop. Vertical and Horizontal are also portuguese words.

Comment (JCKodel) [reply lib/src/l10n/languages/croppy_pt.dart]: I've changed the file, but not sure how to resolve it. 

https://github.com/kekland/croppy/pull/11/commits/ae86f9f8415af07bc9dd9aade521cf345640f271

**Comment by JCKodel** [comment]
Fixed in https://github.com/kekland/croppy/pull/11/commits/ae86f9f8415af07bc9dd9aade521cf345640f271

**Comment by kekland** [comment]
Looks great, thanks! Will merge and add attributions to README

**Comment by kekland** [comment]
Deployed in version 1.1.4

**Pull request merged**